src/data/map.js

```diff
@@ -481,6 +486,22 @@ const getPokestops = async (minLat, maxLat, minLon, maxLon, updated = 0, showPok
             excludePokemonSQL = sqlExcludeCreate;
         }
 
+        if (excludedEvolutions.length === 0) {
+            excludeEvolutionSQL = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 12 AND quest_pokemon_id IS NOT NULL)';
+        } else {
+            let sqlExcludeCreate = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 12 AND quest_pokemon_id IS NOT NULL AND quest_pokemon_id NOT IN (';
```

Comment:
```suggestion
        if (excludedEvolutions.length === 0) {
            excludeEvolutionSQL = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 12)';
        } else {
            let sqlExcludeCreate = 'OR (quest_reward_type IS NOT NULL AND quest_reward_type = 12 AND json_extract(json_extract(quest_rewards, "$[*].info.pokemon_id"), "$[0]") NOT IN (';
```

The above suggestion looks correct for me but doesn't seem to work...Not sure what I'm missing.